src/API/src/ingest/ingest.service.ts

```diff
@@ -117,7 +117,6 @@ export class IngestService {
 
       await this.bionomicsRepository.save(bionomicsArray);
       await this.linkOccurrence(bionomicsArray);
-      triggerAllDataCreationHandler();
```

Comment:
Why have you removed this?

Reply:
Because currently we are unsure of how to map the data that the user wishes to download and how bionomics fits in to that. So currently, there is only logic to handle occurrence data and how that feeds into the csv that is built. If bionomics is uploaded it will pass an empty array and fail whilst this is present.

I thought it better to wait until the mapping is finalised before feeding bionomics ingest into it